src/de740_len_23f0.c

```diff
@@ -272,6 +272,12 @@ void create_shading_palette(Matrix4f mtx, s32 uls, s32 ult, s32 lrs, s32 lrt, s3
     }
 }
 
+extern int ENVIRONMENT;
+extern int SHADE;
+extern int COMBINED;
+extern int COMBINED_ALPHA;
+extern int TEXEL0_ALPHA;
```

Comment:
What are these? They don't appear to be used

Reply:
they're to help with the permuter while this function is WIP